### PR TITLE
Devise and Rails Engine 

### PR DESCRIPTION
**Pull request by shivraj-webonise**

### CODE REVIEW

File: blog/app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+         has_many :posts
```
Comment (nischay-webonise) [inline blog/app/models/user.rb:7]: indentation fix

File: blog/app/views/posts/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<%= link_to 'Back', posts_path, class: 'btn btn-secondary', style: 'margin-bottom:50px' %>
+
+
+<div class='row'>
+    <div class= 'col'>
+        <div class='card'>
+            <div class='card-body'>
+            <h4 class='card-title'> Editing Post</h4>
+                                        
+                        <%= form_with(model: @post) do |form| %>
```
Comment (nischay-webonise) [inline blog/app/views/posts/edit.html.erb:10]: Same indentation issues from last  PR

File: blog/app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<div class="posts">
+<% @posts.each do |post| %>
+  <%= link_to post, class:'text-decoration-none' do %>
+    <div class="card mb-3">
+      <div class= "card-body">
+        <div class="row">
+        <div class="col-md-2">
+          <% if post.image_url.present? %>
+              <%= image_tag post.image_url , style: 'height:200px; width: 200px'%>
```
Comment (nischay-webonise) [inline blog/app/views/posts/index.html.erb:23]: indentation issues use 2 space

File: engines/static/app/controllers/static/contacts_controller.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+require_dependency "static/application_controller"
+
+module Static
+  class ContactsController < ApplicationController
```
Comment (nischay-webonise) [inline engines/static/app/controllers/static/contacts_controller.rb:4]: You added this contacts controller, I didn't understand:
You were supposed to add static pages controller for content management.

Comment (shivraj-webonise) [reply engines/static/app/controllers/static/contacts_controller.rb:4]: Company website has a ContactUs Page so tried to create Content Management for ContactUs page. Its somewhat similar to helpdesk which you gave as an example.

File: blog/app/models/user.rb (hunk 1 above)
Comment (shivraj-webonise) [reply blog/app/models/user.rb:7]: I resolved these issues in second PR

File: blog/app/views/posts/edit.html.erb (hunk 2 above)
Comment (shivraj-webonise) [reply blog/app/views/posts/edit.html.erb:10]: Resolved in second PR

File: blog/app/views/posts/index.html.erb (hunk 3 above)
Comment (shivraj-webonise) [reply blog/app/views/posts/index.html.erb:23]: Resolved in second PR

**Review by nischay-webonise — APPROVED** [review]

**Pull request merged**